Config API/Count Channels and Devices/README.md

```diff
@@ -0,0 +1,20 @@
+# Count the number of channels and devices in a Kepware Server Project
+
+## Install Python and the Kepware Config API Python SDK
+
+1. Download [Python](https://www.python.org/downloads/)
+2. During the install, check Add Python 3.x to PATH
+3. Download and extract Kepware Config API Python SDK from https://github.com/PTCInc/Kepware-ConfigAPI-SDK-Python
```

Comment:
```suggestion
3. Install the [Kepware Config API SDK](https://github.com/PTCInc/Kepware-ConfigAPI-SDK-Python) library
```
Installation instructions already exist on the reference page. No need to repeat here.